StripePayments/StripePayments/Source/Helpers/STPBankAccountCollector.swift

```diff
@@ -110,6 +110,35 @@ public class STPBankAccountCollector: NSObject {
         params: STPCollectBankAccountParams,
         from viewController: UIViewController,
         completion: @escaping STPCollectBankAccountForPaymentCompletionBlock
+    ) {
+        collectBankAccountForPayment(
+            clientSecret: clientSecret,
+            returnURL: returnURL,
+            params: params,
+            from: viewController,
+            onEvent: nil,
+            completion: completion
+        )
+    }
+
+    /// Presents a modal from the viewController to collect bank account
+    /// and if completed successfully, link your bank account to a PaymentIntent
+    /// - Parameters:
+    ///   - clientSecret:      Client secret of the payment intent
+    ///   - returnURL:         A URL that redirects back to your app to be used to return after completing authentication in another app (such as bank app or Safari).
+    ///   - onEvent:           The `onEvent` closure is triggered upon the occurrence of specific events during the process of a user connecting their financial accounts.
```

Comment:
nit: move `onEvent` to after `viewController`

Reply:
ty nice catch; fixed in commit coming soon